src/Integration.Vsix/Commands/HelpMenu/HelpMenu.vsct

```diff
@@ -9,10 +11,12 @@
     </Groups>
 
     <Buttons>
-      <Button guid="HelpMenuCmdSet" id="ReportProblemCommand" priority="0x100">
-        <Parent guid="HelpMenuCmdSet" id="HelpGroup"/>
+      <Button guid="HelpMenuCmdSet" id="ViewDocumentationCommand" priority="0x100">
+        <Parent guid="HelpMenuCmdSet" id="HelpGroup" />
+        <Icon guid="ImageCatalogGuid" id="StatusHelp" />
+        <CommandFlag>IconIsMoniker</CommandFlag>
         <Strings>
-          <ButtonText>Report a problem</ButtonText>
+          <ButtonText>View Documentation</ButtonText>
```

Comment:
Looking at resharper and VS, the help command was always first and with an icon, so I moved it higher
![image](https://user-images.githubusercontent.com/60586879/206154338-503ff200-6ab4-45c8-b864-5743c049d83f.png)